src/main/java/hwr/oop/Priority.java

```diff
@@ -1,20 +1,17 @@
 package hwr.oop;
 
 public enum Priority {
-    LOW,
-    MEDIUM,
-    HIGH;
+    LOW(0),
+    MEDIUM(1),
+    HIGH(2);
 
+    Priority(int value) {
+        this.value = value;
+    }
+
+    private int value;
```

Comment:
zwischen 
```Java
private int value;
public int toInt() {
...
```
Bitte eine Zeile freilassen:
```suggestion
private int value;

public int toInt() {
...
```